x-pack/plugin/rank-rrf/src/yamlRestTest/resources/rest-api-spec/test/linear/10_linear_retriever.yml

```
@@ -1333,3 +1333,605 @@ setup:
+
+---
+"linear retriever with top-level normalizer - empty results":
+  - requires:
+      cluster_features: [ "linear_retriever.top_level_normalizer" ]
+      reason: "Support for top-level normalizer in linear retriever"
+  - do:
+      search:
+        index: test
+        body:
+          retriever:
+            linear:
+              normalizer: minmax
+              retrievers: [
+                {
+                  retriever: {
+                    standard: {
+                      query: {
+                        term: {
+                          keyword: {
+                            value: "nonexistent"
+                          }
+                        }
+                      }
+                    }
+                  },
+                  weight: 1.0
+                }
+              ]
+
+  - match: { hits.total.value: 0 }
+  - length: { hits.hits: 0 }
+
+---
+"Linear retriever with per-retriever normalizer override":
+  - requires:
+      cluster_features: [ "linear_retriever.l2_norm", "linear_retriever.top_level_normalizer" ]
+      reason: "Support for L2 normalization and top-level normalizer in linear retriever"
+  - do:
+      search:
+        index: test
+        body:
+          retriever:
+            linear:
+              normalizer: minmax
+              retrievers: [
+                {
+                  retriever: {
+                    standard: {
+                      query: {
+                        constant_score: {
+                          filter: {
+                            term: {
+                              keyword: {
+                                value: "one"
+                              }
+                            }
+                          },
+                          boost: 5.0
+                        }
+                      }
+                    }
+                  },
+                  weight: 1.0,
+                  normalizer: l2_norm
+                },
+                {
+                  retriever: {
+                    standard: {
+                      query: {
+                        constant_score: {
+                          filter: {
+                            term: {
+                              keyword: {
+                                value: "four"
+                              }
+                            }
+                          },
+                          boost: 12.0
+                        }
+                      }
+                    }
+                  },
+                  weight: 1.0
+                }
+              ]
+
+  - match: { hits.total.value: 2 }
+  - match: { hits.hits.0._id: "1" }
+  - match: { hits.hits.1._id: "4" }
+  - close_to: { hits.hits.0._score: { value: 1.0, error: 0.001} }
+  - close_to: { hits.hits.1._score: { value: 1.0, error: 0.001} }
+
+---
+"Linear retriever with mixed normalizers - inheritance test":
+  - requires:
+      cluster_features: [ "linear_retriever.l2_norm" ]
```

Don't we need to specify the top level normalizer feature? 
```suggestion
      cluster_features: [ "linear_retriever.top_level_normalizer" ]
```